packages/coingecko/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@usedapp/coingecko",
+  "version": "0.3.21",
+  "main": "dist/cjs/src/index.js",
+  "module": "dist/esm/src/index.js",
+  "types": "dist/esm/src/index.d.ts",
+  "repository": "git@github.com:EthWorks/useDApp.git",
+  "author": "Ethworks",
+  "license": "MIT",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.2.14",
+    "@types/chai-as-promised": "^7.1.3",
+    "@types/mocha": "^8.2.0",
+    "@typescript-eslint/eslint-plugin": "^4.10.0",
+    "@typescript-eslint/parser": "^4.10.0",
+    "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
+    "eslint": "7.19.0",
+    "mocha": "^8.2.1",
+    "prettier": "^2.1.2",
+    "ts-node": "^9.1.1",
+    "typescript": "^4.1.3"
+  },
+  "dependencies": {
+    "@types/react": "17.0.1",
+    "@usedapp/core": "^0.3.21",
+    "whatwg-fetch": "^3.6.2"
+  },
+  "peerDependencies": {
+    "react": "*"
+  },
```

Comment:
I don't think we need peerDependency here, we can just use React that was resolved from @usedapp/core dependency